src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
```

Comment:
The class RightTriangle should not start with an empty line after the class declaration. Please remove the empty line to adhere to Java best practices.